modules/configuration-ovnk-network-plugin-json-object.adoc

```diff
@@ -38,6 +38,8 @@ For `"topology":"layer2"` deployments, IPv6 (`2001:DBB::/64`) and dual-stack (`1
 |`netAttachDefName`
 |`string`
 | The namespace and the network attachment name. The value must match the values specified for `namespace` and `name` when defining the network attachment definition object. For example, `ns1/l2-network` refers to the `ns1` namespace and the `l2-network` network attachment definition name.
++
+The value must match the `namespace` and metadata name of the network attachment definition object where this configuration is included.
```

Comment:
I just notice that are just saying the same thing in line 40 above. I prefer this version but embed in the paragraph. 

So I would rephrase the whole thing to: 
```
The metadata `namespace` and `name` of the network attachment definition object where this
configuration is included. For example, if this configuration is defined in a network attachment
definition in namespace `ns1` named `l2-network`, this should be set to `ns1/l2-network`.
```